### PR TITLE
Animated coin flip

### PR DESCRIPTION
**Pull request by quardbreak**
https://streamable.com/84k8nu

### CODE REVIEW

**Review by MistakeNot4892 — DISMISSED** [review]

**Comment by MistakeNot4892** [comment]
Code is more or less fine but the actual animation is really quite ugly I think. Not sold on it.

**Review by MistakeNot4892 — APPROVED** [review]
Eh why not.

File: code/game/objects/items/weapons/material/coins.dm [hunk 1]
```diff
@@ -58,29 +58,69 @@
...
+	flipit.Scale(0.2, 1)
+	animate(src, transform = flipit, time = 2, easing = QUAD_EASING)
+
+	flipit.Scale(5, 1)
+	if(pick(0, 1))
+		flipit.Invert()
+	flipit.Turn(rand(1, 359))
+	animate(src, transform = flipit, time = 2, easing = QUAD_EASING)
+
+	if (prob(0.1) || rigged)
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/material/coins.dm:97]: Doesn't this mean a coin will now always land on the edge forever?

Comment (quardbreak) [reply code/game/objects/items/weapons/material/coins.dm:97]: nope, not always. `rigged` turned off by default and needed for special actions like cheating :^)

Comment (MistakeNot4892) [reply code/game/objects/items/weapons/material/coins.dm:97]: Oh, this is for a future feature? I didn't see `rigged` was an argument not a member variable.

**Pull request merged**